### ISSUE TITLE
Misc Questions

### ISSUE THREAD

**Issue by GaryFurash** [Laverna/laverna#398]
1. Where can we ask questions. Issues seems more like it's for things that are broken.
2. How do you use TAGS. I've made some, but have no idea how to associate them with anything.
3. Is there a way to import existing files and stuff that was not created by Laverna?
4. Can I just send someone money through PayPal. I don't want to create a bitcoin account.
5. Thanks

**Comment by wwebfor** [comment]
1. You can ask questions on Twitter (@lavernaApp), IRC (#laverna),  [Gitter][1], or [email][2]
2. You can create or attach tags by using hashtags. For example, if you write in a note `#laverna`, it will attach a tag (in this case `laverna`). If a tag doesn't exist, it will automatically create it.
3. Right now there is no mechanism to import files not created in the app itself, but in the future we may add it.
4. Unfortunately we can't accept donates through Paypal.
   If you want to donate to accelerate the overall development of the project, you can do it on [our Salt page on BountySource][3].
   If you want to accelerate the development of a particular issue, you can post a bounty on [BountySource][1].
5. Thank you for your interest in Laverna!

[1]: https://gitter.im/Laverna/laverna
[2]: mailto:lavernaproject@gmail.com
[3]: https://www.bountysource.com/teams/laverna

**Comment by tjadevries** [comment]
I had the same question about tags and fortunately found the the answer here. Twitter etc. are not really useful for "tips and tricks"; I would suggest the [wiki](https://github.com/Laverna/laverna/wiki). Agree?

**Comment by wwwredfish** [comment]
@tjadevries , Yes, you are right. We really need to document everything... We will try to do that if/when we have some free time.

**Comment by tjadevries** [comment]
Suggestion: paste the contents of the attached file in a new wiki page



2016-02-28 15:32 GMT+01:00 wwwredfish <notifications@github.com>:
